src/main/java/org/example/homework6/Task5.java

```diff
@@ -0,0 +1,46 @@
+package org.example.homework6;
+
+import java.util.Scanner;
+
+public class Task5 {
+    public static void main(String[] args) {
+        double a;
+        double b;
+        char op;
+        Scanner ab = new Scanner(System.in);
+        while (true) {
+            System.out.print("Enter a number: ");
+            a = Double.parseDouble(ab.nextLine());
+            System.out.print("Enter an operation (+, -, *, /): ");
+            op = ab.nextLine().trim().toCharArray()[0];
+            System.out.print("Enter another number: ");
+            b = Double.parseDouble(ab.nextLine());
+            switch (op) {
```

Comment:
You can extract switch to separate method

Reply:
Done